OTTheme/aliases.yml

```diff
@@ -13,6 +13,10 @@ aliases:
     value: "#E5656E"
   color-black-cherry:
     value: "#811520"
+  color-warning-red:
+    value: "#EFD4D4"
+  color-content-highlighted-red:
+    value: "#FEF8F0"
```

Comment:
People seem to dislike functional color naming in https://github.com/opentable/design-tokens/issues/52